spa/src/utils/index.ts

```diff
@@ -8,6 +10,14 @@ export function popup (url: string, opts: { width: number, height: number }) {
 	return window.open(url, "_blank", `popup,${popup_params(opts.width, opts.height)}`);
 }
 
+export function reportError(err: unknown) {
+	try {
+		Sentry.captureException(err);
+	} catch (_) {
+		//do nothing
```

Comment:
/nit Do we want to log this out for us still, in case Sentry has a cry about something

```
catch (err) {
 console.error("Sentry failed blah blah blah", error)
}
```

Reply:
Not sure, in fact, in local, since sentry will not be set, nothing will happen ( and no error thrown anyway )
So unless you grab the (prod) sentry token in set it in your local env, you won't see a thing as sentry will NOT be inited locally.